.github/workflows/go.yml

```diff
@@ -17,6 +17,9 @@ jobs:
         with:
           go-version: 1.17
 
+      - name: Lint
```

Comment:
We will need a prior step to install the linter, similar to how Go was installed.

Reply:
A step like this can be added.

``` yaml
      - name: golangci-lint
        uses: golangci/golangci-lint-action@v2
        with:
          version: v1.29
```

From https://github.com/golangci/golangci-lint-action#how-to-use

Reply:
ha... right!    I have deep exp with circleci, travis, jenkins etc.   but new to github workflows...   it's worth a eagle eye review.   The docs recommend a separate workflow job which I agree with. (jobs are run in parallel).  But configured based on guidance.   Ready!